.github/workflows/builds.yml

```
@@ -0,0 +1,119 @@
+  APPLE_TEAM_ID: ${{ secrets.APPLE_TEAM_ID}}
+  FASTLANE_ITC_TEAM_ID: ${{ secrets.APPLE_TEAM_ITC_ID}}
+  FASTLANE_USER: ${{ secrets.FASTLANE_USER }}
+  FASTLANE_PASSWORD: ${{ secrets.FASTLANE_PASSWORD }}  
+  FASTLANE_APPLE_APPLICATION_SPECIFIC_PASSWORD: ${{ secrets.FASTLANE_APPLE_APPLICATION_SPECIFIC_PASSWORD }}
+  # Android Release
+  JSON_KEYFILE: api-rootstrap.json
+  RELEASE_STORE_FILE: debug.keystore
+  RELEASE_STORE_PASSWORD: ${{ secrets.ANDROID_RELEASE_STORE_PASSWORD }}
+  RELEASE_KEY_ALIAS: debug
+  RELEASE_KEY_PASSWORD: ${{ secrets.ANDROID_RELEASE_KEY_PASSWORD }}
+  # Notifications
+  SLACK_URL: ${{ secrets.SLACK_URL }}
+  SLACK_CHANNEL: '#dev-builds'
+
+jobs:
+
+  ios:
+    runs-on: macmini
+    timeout-minutes: 45
+
+    steps:
+   
+    - name: Checkout
+      uses: actions/checkout@v2
+        
+    # This needs to match Node version in package.json
+    - uses: actions/setup-node@v1
+      with:
+        node-version: '14.5.0'
+    
+    
+    # Downloads certificate, private key and Firebase file
+    - name: Download code signing items
+      run: | 
+        aws s3 cp s3://$KEYS_BUCKET/$FOLDER/ ./ios --recursive
+        mkdir -p ~/Library/MobileDevice/Provisioning\ Profiles
+        mv ./ios/$APPLE_PROFILE ~/Library/MobileDevice/Provisioning\ Profiles/$APPLE_PROFILE
+    
+    - name: install Node dependencies
+      run: yarn install
+
+    - name: Generate .env file
+      run: |
+        echo API_URL=$API_URL >> .env.dev
+
+    # Runs build and archives for AdHoc distribution
+    # Pushes build to AWS S3
+    # Sends Slack notification
+    - name: Build with Fastlane
+      run: bundle exec fastlane share_develop
+      working-directory: ios
+
+
+  android:
+    runs-on: ubuntu-latest
+    timeout-minutes: 45
+
+    steps:
+    
+    - name: Checkout
+      uses: actions/checkout@v2
+    
+    # This needs to match Node version in package.json
+    - uses: actions/setup-node@v1
+      with:
+        node-version: '14.5.0'
+
+    # Downloads certificate, private key and Firebase file
+    - name: Download code signing items
+      run: | 
+        aws s3 cp s3://$KEYS_BUCKET/$FOLDER/ ./android --recursive
+        mv ./android/$RELEASE_STORE_FILE ./android/app/$RELEASE_STORE_FILE
+
+    - name: install Node dependencies
+      run: yarn install
+
+    - name: Generate .env file
+      run: |
+        echo API_URL=$API_URL >> .env.dev
+        echo RELEASE_STORE_FILE=$RELEASE_STORE_FILE >> .env.dev
+        echo RELEASE_STORE_PASSWORD=$RELEASE_STORE_PASSWORD >> .env.dev
+        echo RELEASE_KEY_ALIAS=$RELEASE_KEY_ALIAS >> .env.dev
+        echo RELEASE_KEY_PASSWORD=$RELEASE_KEY_PASSWORD >> .env.dev
+
+    - name: install gem dependencies
+      run: bundle update
+      working-directory: android
+    # Cleans workspace
+    # Runs build with Gradle
+    # Submits build to Play Store
+    # Sends Slack notification
+    - name: Build with Fastlane
+      run: bundle exec fastlane release_dev_appstore
```

That lane does not exist
```suggestion
      run: bundle exec fastlane release_develop
```

---

thanks for all the catches!  happens when propagating changes across repos back and forth. Should all be fixed now